src/components/interfaces/MOBILE_API.xml

```diff
@@ -2172,6 +2175,7 @@
         </param>
         <param name="hmiPermissions" type="HMIPermissions"  mandatory="true"/>
         <param name="parameterPermissions" type="ParameterPermissions"  mandatory="true"/>
+        <param name="requireEncryption" type="Boolean" mandatory="false" since="5.1"/>
```

Comment:
```suggestion
        <param name="requireEncryption" type="Boolean" mandatory="false" since="6.0"/>
```

Reply:
fixed in 45f0888b531adde8ac91715a3c526c75c7c23ba3